scripts/agentica/patterns/chain_of_responsibility.py

```
@@ -0,0 +1,134 @@
+
+        Returns:
+            Result from the handler that processed the request
+
+        Raises:
+            ValueError: If no handler can handle the query
+        """
+        # Generate unique chain ID for this execution
+        cor_id = f"cor-{uuid4()}"
+
+        # Find first handler that can handle the query
+        for idx, handler in enumerate(self.handlers):
```

Ruff (0.14.10)

Loop control variable `idx` not used within loop body

Rename unused `idx` to `_idx`

(B007)